src/app/public/modules/type-definitions/type-anchor-links.pipe.ts

```diff
@@ -17,8 +17,11 @@ export class SkyDocsTypeAnchorLinksPipe implements PipeTransform {
     private anchorLinkService: SkyDocsAnchorLinkService
   ) { }
 
-  public transform(value: string): string {
-    return this.anchorLinkService.applyTypeAnchorLinks(value);
+  public transform(
+    value: string,
+    addCodeFormatting?: boolean
```

Comment:
What if we went with a string enum for the formatting type so that the pipe (when written out) made sense without looking at the docs? The names aren't great, but something similar to the following?
```
{{ description | skyDocsTypeAnchorLinks:'no-code-tags' }}

public transform(value: string, formatType: SkyDocsAnchorLinkServiceFormatType) {
  const applyCodeFormatting = (formatType === 'no-code-tags');
}
```

Reply:
Yeah, that's a great idea. Much more deliberate. I'll make the change.

Reply:
Done.